library/core/src/hash/mod.rs

```diff
@@ -175,6 +175,21 @@ pub trait Hash {
 
     /// Feeds a slice of this type into the given [`Hasher`].
     ///
+    /// This method is meant as a convenience, but its implementation is
+    /// also explicitly left undefined. It isn't guaranteed to be
```

Comment:
```suggestion
    /// also explicitly left unspecified. It isn't guaranteed to be
```
Undefined behavior means "the compiler can delete large swaths of code", not "the implementation can return an arbitrary value". https://stackoverflow.com/a/2397995/7669110